src/scss/base/_typography.scss

```diff
@@ -60,13 +72,18 @@ h6,
 .epsilon {
     color: $color-headings;
     margin: 0;
-    font-family: $font-family-headings;
-    font-weight: $font-weight-headings;
     margin-bottom: 0;
+    font-family: $font-family-headings;
+    font-weight: $font-weight-base;
```

Comment:
I think this needs to be `$font-weight-headings`. It's messing up the heading weights for me.